### PR TITLE
check exportability

### PR DESCRIPTION
**Pull request by jackie-pc**

### CODE REVIEW

File: .github/workflows/check_export.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+              fi
+              python -m venv venv
+              source venv/bin/activate
+              # pin reflex version - no moving target
+              pip install reflex==0.2.7
+              pip install -r requirements.txt
+              reflex init
+              reflex export
+              deactivate
+              if ls frontend.zip backend.zip; then
```
Comment (masenf) [inline .github/workflows/check_export.yml:30]: if possible, we should check that these are valid zips as well

**Review by masenf — APPROVED** [review]
is the backend.zip supposed to contain the entire virtualenv?

File: .github/workflows/check_export.yml (hunk 1 above)
Comment (jackie-pc) [reply .github/workflows/check_export.yml]: Sure doing `unzip -t` is easy enough.

**Comment by jackie-pc** [comment]
> 

Nope - it's one of the shortcomings of our `reflex export --backend` I was referring to before.  IMO we need to implement smarting default logic plus some config knobs for overriding.  E.g. for venvs.... we can do some heuristics to detect virtualenv dirs and exclude them, and have an `--include` configurable to bring it back if someone really knows they want it in there.  Tracked [here](https://linear.app/reflex-dev/issue/REF-531/backendzip-construction-should-have-configurable-exclude-mech).

**Pull request merged**